ios/RNAppAuth.m

```diff
@@ -44,12 +44,13 @@ - (dispatch_queue_t)methodQueue
                  subjectType: (NSString *) subjectType
                  tokenEndpointAuthMethod: (NSString *) tokenEndpointAuthMethod
                  additionalParameters: (NSDictionary *_Nullable) additionalParameters
+                 connectionTimeoutSeconds: (double) connectionTimeoutSeconds
```

Comment:
This line appears to have introduced a regression: #790 

Proposed fix is here: #804